Cargo.toml

```diff
@@ -43,6 +43,7 @@ document-features = { version = "0.2.7", optional = true }
 lru = "0.12.0"
 stability = "0.2.0"
 compact_str = "0.7.1"
+unicode-truncate = "1"
```

Comment:
We should probably sort these or something… currently its somewhat of a mess, historically layered or something…

Reply:
5cc0072f

Reply:
Fixed in 5cc0072f6